dali/pipeline/operator/builtin/conditional/split_merge.h

```diff
@@ -30,10 +33,16 @@ namespace dali {
  * @param condition_idx Index of the sample that this condition applies to (input for split, output
  * for merge)
  */
-inline int get_group_index(const TensorList<CPUBackend>& condition, int condition_idx,
-                              bool is_logical = true) {
+inline int get_group_index(const TensorList<CPUBackend> &condition, int condition_idx,
```

Comment:
Just a food for thought, could it affect perf if we accepted TensorView here and did the switch for the while batch instead as we do most of the time (when samples are far from scalars of course).